Readme.md

```diff
@@ -191,7 +191,8 @@ while (keys.length) {
 
 [crockfordconvention]: http://javascript.crockford.com/code.html
 
-## Use lowerCamelCase for variables, properties and function names
+## Naming Conventions
+### Use lowerCamelCase for variables, properties and function names
```

Comment:
So... I think this is where we have a divide, because I definitely prefer lowerCamelCase for variables and functions, but I never wanted to force the style. Of all the style choices, I feel like this one impacts so much of what you write, so it's pretty hard when you have a preference that makes it easier for you. I don't know what to do about this one honestly. I guess if we believe standardization matters most to us, then I'd argue for lowerCamelCase but be ok with what we decide together.